lib/rules/no-unknown-arguments-for-builtin-components.js

```diff
@@ -292,28 +292,31 @@ function ERROR_MESSAGE(tagName, argumentName) {
   const pureQuery = removeAtSymbol(argumentName);
   let query = pureQuery;
   let fuzzyResults = [];
+  let hasArgumentsMatch = candidates.includes(pureQuery);
 
   while (!fuzzyResults.length && query.length) {
     fuzzyResults = new Fuse(candidates).search(query);
     query = query.slice(0, -1);
   }
 
-  let closestKey = fuzzyResults.length ? `@${fuzzyResults[0].item}` : argumentName;
-
-  if (closestKey in deprecatedArgs) {
-    return deprecateArgument(tagName, closestKey, deprecatedArgs[closestKey]);
+  if (!hasArgumentsMatch) {
+    const msg = `"${argumentName}" is unknown argument for <${tagName} /> component.`;
```

Comment:
I would consider changing the message to this (and its associated test):

```suggestion
    const msg = `"${argumentName}" is not a known argument for the <${tagName} /> component.`;
```